components/Heroes/HeroesIndex.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { REDUCER_KEY } from './../../reducers';
+import HeroList from './HeroList';
+import style from './heroes.css';
+
+const HeroesIndex = ({ heroes }) => (
+  <div>
+    <h1 className={style.Header}>All Dota 2 Heroes</h1>
+    <HeroList heroes={heroes} />
+  </div>
+);
+
+const mapStateToProps = (state) => ({
+  heroes: state[REDUCER_KEY].heroes,
+});
```

Comment:
I understood what '/reducers/gotPlayer/matches.js' does. But I don't understand what do you mean by "accept state as parameter". Can you give me some example?